package.json

```diff
@@ -10,8 +10,9 @@
   "homepage": "https://github.com/UKHomeOffice/hof-controllers",
   "author": "HomeOffice",
   "scripts": {
-    "test": "NODE_ENV=test mocha",
+    "test": "npm run lint && npm run style && npm run mocha",
```

Comment:
This is probably just personal preference, but I'd leave `NODE_ENV` in `test`since this is where I'd expect to see it.